assets/js/vue-apps/grant-detail.js

```diff
@@ -8,27 +9,26 @@ function init() {
     }
     new Vue({
         el: mountEl,
-        delimiters: ['<%', '%>'],
         data() {
             // Populate data from global object (eg. to share server/client state)
             let PGS = window._PAST_GRANTS_SEARCH;
             return {
                 grant: PGS.project,
-                relatedGrants: false
+                relatedGrants: null
             };
         },
         mounted: function() {
             let geocode;
             if (this.grant.beneficiaryLocation) {
-                let loc = this.grant.beneficiaryLocation.find(l => l.geoCodeType === 'CMLAD');
+                let loc = find(this.grant.beneficiaryLocation, location => location.geoCodeType === 'CMLAD');
```

Comment:
Out of interest, why load in a module to do this instead of using the native version?

Reply:
Because IE 11 doesn't support `Array.find`